### DISCUSSION TITLE
QnA

### DISCUSSION THREAD

**Discussion by vaishnavi-3969** [vaishnavi-3969/Web-Componentify#13] [Q&A] [answered]
Doubts/Concerns

**Comment by AthulKkumar** [comment]
I didn't get what  isweb componentify and also the footer alignments are messed up.

**Reply by vaishnavi-3969** [reply, marked as answer]
Web Componentify is an open-source repository dedicated to simplifying web development by providing a collection of reusable components for building a website, web application, or mobile app, you'll find a wide range of components, from headers and footers to buttons and dropdowns, available in various popular frameworks and language

**Reply by vaishnavi-3969** [reply]
Regarding the footer will look into that thanks for the feedback 😄
